exoscale/resource_exoscale_network.go

```diff
@@ -23,10 +23,6 @@ func resourceNetwork() *schema.Resource {
 			Required: true,
 			ForceNew: true,
 		},
-		"network_offering": {
```

Comment:
You should actually leave the attribute and add replace `Required: true` with `Removed: true`.

Reply:
Better than removing attribute savagely 😆

Reply:
After discussion: even better is to deprecate the attribute.